pages/docs/tutorials/multiplatform-library.md

```diff
@@ -190,18 +190,18 @@ object NativeBase64Encoder : Base64Encoder {
 ```
 </div>
 
-Now we have implementations on all platforms and it is time to move to testing of our library.
+Now we have implementations on all the platforms and it is time to move to testing of our library.
 
 # Testing
 
-To make library complete we should write some tests, but we have three independent implementations and it will be a waste to write duplicates tests for each one.
-Good thing about common code is that it can be covered with common tests, which later are compiled and executed on *every* platform.
- All the bits for testing are already generated by the project Wizard. TODO link to Sergei article to avoid explanation of `test` source sets and dependencies.
+To make the library complete we should write some tests, but we have three independent implementations and it is a waste of time to write duplicate tests for each one.
+The good thing about common code is that it can be covered with common tests, which later are compiled and executed on *every* platform.
+All the bits for testing are already generated by the project Wizard. TODO link to Sergei article to avoid explanation of `test` source sets and dependencies.
 
-Let's create class `Base64Test` in `commonTest/kotlin/jetbrains/base64` folder and write a basic tests for Base64.
+Let's create the class `Base64Test` in `commonTest/kotlin/jetbrains/base64` folder and write the basic tests for Base64.
 
-But as you remember, our API converts byte arrays to byte arrays and it's not quite handy to test byte arrays.
-So before we start writing a test, let's add a method `encodeToString` with a default implementation to our `Base64Encoder` interface:
+But as you remember, our API converts byte arrays to byte arrays and it's not useful to test byte arrays.
```

Comment:
converts bit arrays to bit arrays?

Reply:
Yes. Will rephrase as "But as you remember, our API converts byte arrays to byte arrays in a different format" if you don't mind.

Also, why is it "not useful"? It is useful, but tests are not human-readable and that's what I mean by "not quite handy".